packages/mui-material/src/styles/createGetSelector.ts

```diff
@@ -33,7 +33,9 @@ export default <
         if (rule === 'media') {
           return {
             ':root': css,
-            '@media (prefers-color-scheme: dark) { :root': excludedVariables,
```

Comment:
This works with `@emotion/react`, but not with `@emotion/serialize` that Pigment CSS is using. May be due to `stylis`.